### ISSUE TITLE
Unload PDFs in separate windows?

### ISSUE THREAD

**Issue by dstillman** [zotero/zotero#3932]
https://forums.zotero.org/discussion/comment/460210/#Comment_460210

Maybe some clever way to do this?

**Comment by mrtcode** [comment]
Yes, I was thinking for a while about moving tab unloading into Zotero Reader. Instead of roughly killing the tab and replacing it with a placeholder it would only unload most of the content inside Zotero Reader iframe.
